packages/terra-toggle/package.json

```diff
@@ -43,5 +43,13 @@
     "wdio-lowlight": "cd ../.. && terra wdio --themes clinical-lowlight-theme",
     "wdio-fusion": "cd ../.. && terra wdio --themes orion-fusion-theme",
     "wdio": "npm run wdio-default && npm run wdio-lowlight && npm run wdio-fusion"
-  }
+  },
+  "files": [
```

Comment:
There is a docs folder under this component and some others as well with Readme and upgrade guides. Should those be included as well?

Reply:
That's a good catch! Looks like those files already exist in terra-core-docs so they can be excluded from here:

https://github.com/cerner/terra-core/tree/main/packages/terra-core-docs/src/terra-dev-site/doc/toggle